VTEX - Shopper Data API.json

```diff
@@ -0,0 +1,718 @@
+                                                                "items": {
+                                                                    "type": "string"
+                                                                }
+                                                            }
+                                                        },
+                                                        "additionalProperties": false
+                                                    }
+                                                }
+                                            },
+                                            "additionalProperties": true,
+                                            "required": [
+                                                "userId",
+                                                "firstName",
+                                                "lastName"
+                                            ],
+                                            "v-indexed": [
+                                                "userId",
+                                                "unitId",
+                                                "firstName",
+                                                "lastName",
+                                                "document",
+                                                "phone"
+                                            ]
+                                        }
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                },
+                "security": [
+                    {
+                        "VtexIdclientAutCookie": []
+                    }
+                ]
+            }
+        },
+        "/api/dataentities/shopper/documents": {
+            "post": {
+                "summary": "Create shopper document",
+                "description": "This endpoint creates a shopper entity document.\r\n\r\n> ℹ️ The shopper schema supports additional properties. Clients can include extra properties (e.g., `phone2`, `marketingOptIn`) in the request body, and they will be persisted alongside the standard fields.## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
+                "tags": ["Shopper documents"],
+                "parameters": [
+                    {
+                        "name": "_schema",
+                        "in": "query",
+                        "required": true,
+                        "schema": {
+                            "type": "string",
+                            "default": "v1"
+                        },
+                        "description": "Schema name. The value must be `v1`."
```

Comment:
If the product requires this value to be `v1`, I think we should consider hardcoding it into the path above. I don't think we have ever done this, but it sounds like it could prevent mistakes. Also not sure whether open api supports querystrings hardcoded in the path.

I noticed that Get shopper document by user ID doesn't require that query string, but the Search endpoint requires it to be v1 as well.